.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
Interesting. This feels like duplication with `prettier`. Instead of doing this, maybe we there's a way to configure `prettier` in the `package.json` or project dot file in order for editors to leverage that?